server/server.go

```diff
@@ -202,6 +198,20 @@ func (m *Command) SetupServer() error {
 	return nil
 }
 
+// GetLogWriter opens a file for logging, or a default io.Writer (such as stderr) for an empty path.
+func GetLogWriter(path string, defaultWriter io.Writer) (io.Writer, error) {
```

Comment:
I think I prefer `GetLogWriter` since this function isn't super general (fixed file mode/permissions, etc)